variables.tf

```diff
@@ -12,6 +12,30 @@ variable "name" {
   description = "The name to give the IBM Event Notification instance created by this module."
 }
 
+variable "destination_name" {
+  type        = string
+  description = "The Destintion name."
+  default     = null
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The bucket name in IBM cloud object storage instance."
```

Comment:
The name of an existing IBM Cloud Object Storage bucket which will be used for storage of failed delivery events.